server/src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -923,6 +924,74 @@ public void nlerp(Quaternion q2, float blend) {
 		normalizeLocal();
 	}
 
+	/**
+	 * Sets the values of this quaternion to the unit average of the given
+	 * Quaternions and return itself
+	 *
+	 * @param qn list of all the Quaternions to average
+	 * @return a new Quaternion resulting from the average of the given
+	 * Quaternions
+	 */
+	public Quaternion fromAveragedQuaternions(AbstractList<Quaternion> qn) {
+		if (qn.size() == 0) {
+			throw new IllegalArgumentException("qn's length can't be 0'");
```

Comment:
Is this supposed to be here?
```suggestion
			throw new IllegalArgumentException("qn's length can't be 0");
```

Reply:
Oops, I blame IDEA for putting 2 ' when I put one...